pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -122,6 +128,16 @@ func (h *NetworkUtilsHandler) AddrAdd(link netlink.Link, addr *netlink.Addr) err
 func (h *NetworkUtilsHandler) LinkSetMaster(link netlink.Link, master *netlink.Bridge) error {
 	return netlink.LinkSetMaster(link, master)
 }
+func (h *NetworkUtilsHandler) UseIptables() error {
```

Comment:
returning an error for a function that gives a binary answer is not ideal. consider returning a `bool` instead.

Reply:
done